### ISSUE TITLE
Pretrain MobileFaceNet 

### ISSUE THREAD

**Issue by namkuner** [deepinsight/insightface#2436]
Could you give me with the pre-trained MobileFaceNet on the WebFace_600k dataset? I really need it. Thank u so much.

**Comment by namkuner** [comment]
@nttstar please...

**Comment by ndcuong91** [comment]
it's in buffalo_sc package here: [https://github.com/deepinsight/insightface/releases](url)

**Comment by hzy459176895** [comment]
@ndcuong91  
The downloaded model appears to be in ONNX format. Do you know where I can find the .pth  file

**Comment by qiu-pinggaizi** [comment]
Hi! We have a fork of arcface_torch that includes multiple MobileFaceNet variants trained on Glint360K:

- **mbf_v3** (3.7M params, 1.15G FLOPs) - lightweight version with adjusted scale/blocks
- **mbf_v3_se** (3.9M params) - with SE attention module, better accuracy
- **mbf_large** (6.3M params) - the larger variant

All are trained on Glint360K (360K identities, 17M images) with verification results on LFW (99.83%), CFP-FP (98.71%), AgeDB-30 (98.05%), etc.

We also implemented a complete compression pipeline: lightweight model design → knowledge distillation → structured pruning (20% channel pruning, final model 11MB).

The pretrained configs and training scripts are available at: https://github.com/qiu-pinggaizi/arcface_torch

Hope this helps!
